### DISCUSSION TITLE
Eventero Weekly Scrum September 27

### DISCUSSION THREAD

**Discussion by Adrianortiz-PR** [uprm-inso4101-2024-2025-s1/semester-project-campus-resource-and-event-management#168] [General]
we recognize that there’s been quite a bit of restructuring within the organization recently. Because of this, we understand that there might not be a lot of tangible work progress to report for this week.

However, we encourage everyone to share a general progress report of what you've been working on, no matter how small. Additionally, feel free to share any thoughts or feelings you have about the recent organizational changes

**Comment by Ojani** [comment]
I created 4 main issues, each with multiple sub issues for the calendar project. The sub issues are yet to have a description assigned to them, but the main issues are fully fleshed out.

**Comment by estefania-2817** [comment]
I created the front end for the signup page, including a page for selection if you want to create a student account or organization account, and a signup page for each type of account.

**Comment by LuisJCruz** [comment]
Starting off this milestone I got assigned the redesigning of the footer. Started brainstorming and already finished part of the redesign. Im confident and satisfied with the management changes made.

**Comment by SaucyRexy** [comment]
Got assigned to Documentation Lead. Have created a Main issue along with its side issues centered around updating the README doc with the new features. Also created the milestone 2 documentation issue with its requirements and guidelines

**Comment by AlejandroClaudio** [comment]
Haven't done work this week due other classes projects and an exam. I believe this restructuring will us help be more organized in future milestones.

**Comment by arianrodz21** [comment]
I've cloned the repository and have learned a lot about svelte, but haven't been able to do much due to heavy workload from other classes.

**Comment by cseba11** [comment]
This week I been assign to the Main-Issue Ribbon where I will work on the all visual elements, work on the functionality to the calendar view buttons and the create event button.

**Comment by Ingrid1089** [comment]
This week I started working on my assigned task and attended the weekly meeting.

**Comment by AlexisSantiago21** [comment]
I was working on some issues and participated in the weekly meeting.

**Comment by SammyAlva04** [comment]
I haven't done a lot this past week due to exams, but I participated in this week's meeting and have begun organizing on how I will begin working this sprint.

**Comment by marymarmaldonado** [comment]
Read new assigned issues for the milestone 2 and participated in this week’s meeting.

**Comment by GabyMarr** [comment]
I am satisfied with the recent changes made. Haven’t gotten any work done since my team leader switch to manager. Waiting on instructions from new team leader.

**Comment by Kiarancole** [comment]
Haven’t done much this week other than receiving instructions from my feature’s new team leader, regarding the structuring of our feature for milestone 2, and divided these issues between my teammate and I. Glad about the restructuring!

**Comment by MiguelGibo** [comment]
Due to the restructuring, I haven't achieved much this week.

**Comment by EdzelOrtiz** [comment]
This week, due to the restructuring, I got assigned as Team Leader, I started working on the milestone 2 issues and maintain a good communication between the managers and the developers.

**Comment by KennethSepu1** [comment]
I worked on fixing issues this week

**Comment by AngelUPR** [comment]
Currently working on issue #174

**Comment by ggonza28** [comment]
Currently doing research to categorize associations with majors in different faculties.

**Comment by cseba11** [comment]
Currently working on issue #135

**Comment by SammyAlva04** [comment]
I have been currently working on issue #142 with the intent of finishing it next week.

**Comment by kevin-d-lopez** [comment]
Haven't achieved much due to the change in management but assigned the new task and started to look into creating the skeleton of the tags feature.

**Comment by Mannyrodd101** [comment]
I created both the Profile Icon and the Back To Top Button for the Homepage.
